store/write_transaction_test.cpp

```diff
@@ -137,8 +137,19 @@ ATF_TEST_CASE_BODY(commit__fail)
         backend.database().exec(
             "CREATE TABLE foo ("
             "a REFERENCES env_vars(var_name) DEFERRABLE INITIALLY DEFERRED)");
-        backend.database().exec("INSERT INTO foo VALUES (\"WHAT\")");
+        // For whatever reason, multiple Linux distros seem to execute the
```

Comment:
This looks like something that someone (tm) should do a deep dive into, sometime.

Reply:
Thank you for the reminder! I'm going to file the issue after I merge this change :).
Also, FWIW... MacOS fails in the same way that FreeBSD does. I'll have to check and see if NetBSD is affected by this at all...

Reply:
Opened #240.